### ISSUE TITLE
naver.com: ads

### ISSUE THREAD

**Issue by JamesKaret** [uBlockOrigin/uAssets#16788]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.naver.com/`

### Category

ads

### Description

It's Korean site.
Used to work just fine but not now for some reason.

I marked the ads sections.
Also, the very top side wasn't blocked before but can you also block this kind of popup, too?

Reproduce:
- Close every chrome windows if they're opened.
- Open chrome.
- Enter URL (`naver.com` or `www.naver.com`)

Expectation:
No ads are shown

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

[capture 2023-02-17 103008](https://user-images.githubusercontent.com/10825448/219526482-cce84344-a7eb-46ca-8688-1abe2d9c767b.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Chromium: 110
filterset (summary): 
  network: 82992
  cosmetic: 38461
  scriptlet: 16933
  html: 0
listset (total-discarded, last-updated): 
  removed: 
    KOR-1: null
  default: 
    user-filters: 0-0, never
    easylist: 61435-45, 4d.1h.31m
    easyprivacy: 30954-1173, 4d.1h.30m
    plowe-0: 3667-899, 10d.1h.7m
    ublock-abuse: 77-0, 4d.1h.35m
    ublock-badware: 4353-120, 59m
    ublock-filters: 32116-133, 58m
    ublock-privacy: 268-1, 4d.1h.34m
    ublock-quick-fixes: 165-21, 17h.23m
    ublock-unbreak: 1887-46, 40m
    urlhaus-1: 6076-0, 17h.22m
filterset (user): [empty]
trustedset: 
  added: [array of 2 redacted]
modifiedUserSettings: 
  showIconBadge: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 2032 ms (selfie)
  maxAssetCacheWait: 77 ms
popupPanel: 
  blocked: 4
  blockedDetails: 
    naver.com: 4
```
</details>

**Comment by JamesKaret** [comment]
Sometimes it shows ads, sometimes doesn't. (except the top one.)
I don't know why but it happens more often.

**Comment by stephenhawk8054** [comment]
```yaml
removed: 
    KOR-1: null
```

Can you enable `KOR: List-KR` list again?

**Comment by piquark6046** [comment]
The advertisement must be blocked by List-KR filters list.
